src/issue-filing/github/github-url-rectifier.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export type UrlRectifier = (url: string) => string;
+
+export const rectify: UrlRectifier = (url: string): string => {
+    const ownerAndRepo = new RegExp('^https?://github.com/[^/]+/[^/]+$');
```

Comment:
Including `github.com` in the regular expression means we won't add a missing /issues to a GitHub Enterprise URL (but it also means we won't break a GitHub Enterprise URL that's formatted differently than we expect, which is a nice effect of how the tests are structured).

Reply:
That was part of the rationale. I don't have much info about how github enterprise works so I'm assuming the url can be fully customized, in case, the rectifier will not touch it, it will be up to the user to write down the correct url, which is the same right now on master anyway so this is definitely an improvement.